core/lexicon/en/default.inc.php

```diff
@@ -138,10 +138,10 @@
 $_lang['element_caption_new'] = 'Caption of New Element';
 $_lang['elements'] = 'Elements';
 $_lang['email'] = 'Email';
-$_lang['empty_recycle_bin'] = 'Remove Deleted Resources';
-$_lang['empty_recycle_bin_confirm'] = 'Are you sure you want to completely and permanently remove all deleted Resources? This is irreversible.';
-$_lang['empty_recycle_bin_empty'] = 'There are no deleted Resources to remove.';
-$_lang['empty_recycle_bin_emptied'] = 'All deleted Resources have been permanently removed.';
+$_lang['empty_recycle_bin'] = 'Delete Deleted Resources';
+$_lang['empty_recycle_bin_confirm'] = 'Are you sure you want to completely and permanently delete all deleted Resources? This is irreversible.';
+$_lang['empty_recycle_bin_empty'] = 'There are no deleted Resources to delete.';
+$_lang['empty_recycle_bin_emptied'] = 'All deleted Resources have been permanently deleted.';
```

Comment:
English isn't my first language but I would use "Erase" instead of "Deleted" when talking about deleting/removing `Deleted Resources`. 
A native English speaker's view on this would be great!